tests/unit/test_interaction.py

```diff
@@ -165,3 +213,108 @@ def test_interaction_matches_request_negative(
     _request
 ):
     assert not interaction_all_fields.matches_request(_request)
+
+
+@mark.parametrize(
+    'files, expected',
```

Comment:
expected what? (something like `expected_sequence_number` makes it easier to understand)
Same with `files`, maybe like `interaction_files`.

Reply:
Even `interaction_file_names`

Reply:
Same in the other tests that do this btw

Reply:
Also, nice set of tests ;)